mifosng-android/src/main/java/com/mifos/mifosxdroid/online/clientdetails/ClientDetailsPresenter.java

```diff
@@ -138,6 +138,10 @@ public void onCompleted() {
                     public void onError(Throwable e) {
                         getMvpView().showProgressbar(false);
                         getMvpView().showFetchingError("Client not found.");
+
+                        //IN CASE THERE IS AN ERROR OR NO INTERNET CONNECTION
+                        //show the default client accounts info
+                        getMvpView().showClientAccount(new ClientAccounts());
```

Comment:
We do not have any default client details if the request is failed. remove this line `getMvpView().showClientAccount(new ClientAccounts());`. This is not making any sense.